domains/zero.json

```diff
@@ -0,0 +1,12 @@
+{
+  "description": "My personal developer portfolio.",
+  "repo": "https://github.com/CoderPopCat/coderpopcat.github.io",
+  "owner": {
+    "username": "CoderPopCat",
+    "email": "zerotwo@popcat.xyz",
+    "twitter": "none"
```

Comment:
is `twitter.com/none` your twitter account?

Reply:
it means I have no twitter account

Reply:
then why are you filling it

Reply:
if you dont have any twitter just omit it